types/lodash/fp/convert.d.ts

```diff
@@ -12,4 +12,4 @@ interface Convert {
 }
 
 declare const convert: Convert;
-export = convert;
+export default convert;
```

Comment:
Oh hm, I think this might break things since lodash is really a commonjs module, not an es6 module. It might be better to move `ConvertOptions` into `common.d.ts`, or just re-define it in `fp.d.ts`.